packages/manager/modules/pci/src/projects/project/private-networks/private-networks.service.js

```diff
@@ -19,11 +27,46 @@ export default class {
         serviceName,
       })
       .$promise.then((privateNetworks) =>
-        sortBy(privateNetworks, 'vlanId').filter(
-          ({ status }) => !DELETING_STATUS.includes(status),
+        this.$q.all(
+          map(privateNetworks, (privateNetwork) =>
```

Comment:
prefer using native `Array.map` function.
```suggestion
          privateNetworks.map((privateNetwork) =>
```